remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties

```diff
@@ -124,7 +124,7 @@ local-outbound-connection.property=The XNIO Options that will be used during the
 sasl=The "sasl" element contains the SASL authentication configuration for this connector.
 sasl.add=Adds the SASL authentication configuration for its connector
 sasl.remove=Removes the SASL authentication configuration for its connector
-sasl.include-mechanisms=The optional nested "include-mechanisms" element contains a whitelist of allowed SASL mechanism names. No mechanisms will be allowed which are not present in this list.
+sasl.include-mechanisms=The optional nested "include-mechanisms" element contains a allowlist of allowed SASL mechanism names. No mechanisms will be allowed which are not present in this list.
```

Comment:
s/a allowlist/an allowlist. However, in any case, it results in redundancy.


What about if we use:

"The optional nested "include-mechanisms" element contains a list of allowed SASL mechanism names. No mechanisms will be allowed which are not present in this list"

or

"The optional nested "include-mechanisms" element contains an allowlist of SASL mechanism names. No mechanisms will be allowed which are not present in this list"


@fjuma as a native English speaker, would you mind helping us with this sentence? thanks!

Reply:
@yersan The first option you proposed looks good. I wonder if we could reword the second sentence to make it more clear, e.g.,

"The optional nested "include-mechanisms" element contains a list of allowed SASL mechanism names. Only mechanisms that are present in this list will be allowed."

Reply:
Thanks, Farah. That sounds also more clear to me too, +1 to your proposal.

Reply:
Thanks @fjuma and @yersan, I've updated the PR accordingly.